.github/workflows/build.yaml

```diff
@@ -26,6 +26,8 @@ concurrency:
 
 env:
     CHIP_NO_LOG_TIMESTAMPS: true
+    # XXX: Workaround for https://github.com/actions/cache/issues/1141
+    SEGMENT_DOWNLOAD_TIMEOUT_MINS: 3
```

Comment:
Do we need this in the global env, or just for that one step?  Either way is OK, I guess, as long as nothing else uses this variable...

Reply:
It is needed only for actions/cache, however, such action might be a part of other actions (e.g. python env setup also uses cache). I think that this env name is long enough not to clash with any other env variable. So, setting it in the global scope will mitigate potential issues with actions/cache in other steps (not only bootstrap cache). Also, there will be less boilerplate in case of a new job with bootstrap cache in one of these files.